.gitignore

```diff
@@ -1,5 +1,6 @@
 # Ignore generated schema
 swagger.json
+!docs/docsite/rst/rest_api/_swagger/swagger.json
```

Comment:
@AlanCoding @relrod @thedoubl3j Hey, not sure who to ask but thought I might try my luck.

I've included `swagger.json` under the new rst directory but we probably want to generate it as part of the docs build. I thought maybe we could call the Makefile from tox to generate the schema before doing a sphinx-build. 

It might be easier to just try and show it rather than explain. But if you'd have opinions or ideas on an approach I'd really appreciate hearing. Cheers!

Reply:
i am on board with that, especially if you think that is the path to go. I don't have much of an opinion either way but want it to be easy/as straight forward as we can make it. I know some folks had some thoughts around changing the makefile, @TheRealHaoLiu this might be a good opportunity to prevent creep in the makefile/split it out per directory. what are your thoughts?

Reply:
Thanks @thedoubl3j I've been working through a bunch of changes to the content with @tvo318 it's going well but is going to result in a pretty chonky initial PR.

I'm going to remove the `swagger.json` file from this PR and remove this line from the gitignore file. At least that way we won't add anything we'd need to delete later. Once we get stuff merged we can figure out how to generate the schema with tox or whatever without getting in TVo's way.